CMakeLists.txt

```diff
@@ -173,7 +173,7 @@ endif()
 message("  CMAKE_INSTALL_FULL_INCLUDEDIR           : ${CMAKE_INSTALL_FULL_INCLUDEDIR}")
 message("  CMAKE_INSTALL_FULL_LIBDIR               : ${CMAKE_INSTALL_FULL_LIBDIR}")
 message("  CMAKE_INSTALL_FULL_DATADIR              : ${CMAKE_INSTALL_FULL_DATADIR}")
-message("Enable:")
+message("Enabled project configuration (by default or parameters):")
```

Comment:
I personally don't understand what you mean under this phrase. To me it is misleading. The previous "Enable" is honest and straightforward: here is list of variables and here is their values. Just read this literally. This is not indication whether feature is really enabled or not, that's list of variables. The fact that OCL could be OFF while variable is ON - that's a bug. We need to fix it, not workaround it.